aries-site/src/examples/templates/inline-notifications/InlineFormValidation.js

```diff
@@ -34,6 +35,7 @@ export const FormValidation = () => {
         <Box gap="small" pad={{ top: 'medium' }}>
           <Notification
             status="critical"
+            icon={<CircleInformation />}
```

Comment:
```suggestion
            icon={<CircleInformation height="medium" />}
```

also in general I'm wondering if this is correct -- feels odd to pair circleinformation with critical status. Although, I know we have this pattern elsewhere. Can a ticket be filed to revisit this separately and see if we should remove these overrides? or if they're okay?

Reply:
yeah also would be nice for this to be out of the box if we want `circle icon` to be tied to status Critical?

Reply:
Agreed. Ideally we would have a Form Validation error status, only for inline notifications